api-extractor/src/Extractor.ts

```diff
@@ -93,4 +105,25 @@ export default class Analyzer {
     const lineNumber: number = sourceFile.getLineAndCharacterOfPosition(start).line;
     this.errorHandler(message, sourceFile.fileName, lineNumber);
   }
+
+  /**
+   * Scans for external package api files and loads them into the docItemLoader member before 
+   * any API analyzation begins.
+   * 
+   * @param externalJsonCollectionPath - an absolute path to to the folder that contains all the external 
+   * api json files. 
+   * Ex: if externalJsonPath is './resources', then in that folder
+   * are 'es6-collections.api.json', 'es6-promise.api.json', etc.
+   */
+  public loadExternalPackages(externalJsonCollectionPath: string): void {
+    if (!externalJsonCollectionPath) {
+      return;
+    }
+
+    const files: string[] = fsx.readdirSync(externalJsonCollectionPath);
```

Comment:
What if the filename does not end with "*.api.json"?  For example "thumbs.db" or "es6-promise.api.json.bak"?